Sources/App/Views/PackageController/PackageShow+Model.swift

```diff
@@ -232,6 +231,18 @@ extension PackageShow.Model {
         )
     }
     
+    func archivedListItem() -> Node<HTML.ListContext> {
```

Comment:
Nitpick: Let's keep these ordered in the file the same way they are used in the output, so let's move this up to below the `authorsListItem`.